platforms/vercel-with-redwood/api/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "api",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {
+    "@prisma/cli": "2.7.0-dev.43",
+    "@prisma/client": "2.7.0-dev.43",
```

Comment:
This does not make sense at the same time as `resolutions` I think?

Reply:
Removed.

Reply:
I think that is still required. I [removed it](https://github.com/prisma/e2e-tests/pull/808/commits/3ffefd337e4d5fd0c98b204367feab0347175f15) and the test failed again 

Expected: "2.7.0-dev.43"
Received: "2.6.1"

Bring it back!

Reply:
it's not required: https://github.com/prisma/e2e-tests/blob/dd9d0cbb77cc55ec2f8e5b655c6eb7797f71300e/libraries/nexus-schema/package.json#L12-L31

it may have failed because of something else

Reply:
It failed because the versions didn't match anymore, I can explore this further at a later time. 

The received version was the one that is bundled with Redwood. 

```
Expected: "2.7.0-dev.43"
Received: "2.6.1"
```

Reply:
Interestingly, just resolutions don't work (probably because this is a workspace - I read an issue that in workspaces resolutions don't work yet) but just using deps works https://github.com/prisma/e2e-tests/pull/816